cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskTracker.java

```diff
@@ -46,6 +49,8 @@ public class ExecutionTaskTracker {
   public static final String COMPLETED = "completed";
   public static final String GAUGE_ONGOING_EXECUTION_IN_KAFKA_ASSIGNER_MODE = "ongoing-execution-kafka_assigner";
   public static final String GAUGE_ONGOING_EXECUTION_IN_NON_KAFKA_ASSIGNER_MODE = "ongoing-execution-non_kafka_assigner";
+  public static final String METER_TASK_EXECUTION_RATE = "task-execution-rate";
+  public static final String METER_PARTITION_DATA_MOVEMENT_RATE = "partition-data-movement-rate";
```

Comment:
Can you please add the unit here? `dataToMove` below is in bytes? Please document briefly how to interpret the 5 constituent metrics partition-data-movement-rate.[Count, [1|5|15]MinuteRate, Mean]. 

I see count going into millions and Mean is about 1000 and continuously climbing. The rate should plateau at some point right?

I'm probably missing  the mapping of the Meter to ingraphs chart type.

Reply:
Not sure what do you mean, but these 2 metrics are just meters not gauge. I have 2 other gauges added